sourcecode/apis/contentauthor/app/Console/Commands/H5PLibraryInstall.php

```diff
@@ -0,0 +1,122 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Console\Commands;
+
+use App\Console\Libraries\CliH5pAjax;
+use App\H5PLibrariesHubCache;
+use App\H5POption;
+use Carbon\Carbon;
+use Illuminate\Console\Command;
+
+class H5PLibraryInstall extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'h5p:library-manage
+                                {action   : install, update-cache}
+                                {library? : For install, machinename of library to install or update}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Install or update H5P library from h5p.org Hub, update the local cache of available libraries';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle(): int
+    {
+        $action = $this->argument('action');
+
+        if (!config('h5p.isHubEnabled', false)) {
+            $this->newLine();
+            $this->warn("Use of h5p.org hub is not enabled. To enable set 'H5P_IS_HUB_ENABLED=true' in your environment file");
```

Comment:
Environment files don't exist in production environments.

```suggestion
            $this->warn("Use of h5p.org hub is not enabled. To enable set 'H5P_IS_HUB_ENABLED=true'");
```